src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflow_with_type_and_service.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:framework="http://symfony.com/schema/dic/symfony"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                        http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+    <framework:config>
+        <framework:workflows>
+            <framework:workflow name="my_workflow">
+                <framework:marking-store>
+                    <framework:type>multiple_state</framework:type>
+                    <framework:service>workflow_service</framework:service>
+                </framework:marking-store>
+                <framework:supports>Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\FrameworkExtensionTest</framework:supports>
+                <framework:places>first</framework:places>
+                <framework:places>last</framework:places>
+                <framework:transitions>
```

Comment:
this node should not exist in the XML. It is not consistent with the way things work elsewhere.